src/test/java/org/opensearch/geospatial/ip2geo/processor/Ip2GeoCacheTests.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.geospatial.ip2geo.processor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.opensearch.OpenSearchException;
+import org.opensearch.test.OpenSearchTestCase;
+
+public class Ip2GeoCacheTests extends OpenSearchTestCase {
+    public void testCachesAndEvictsResults() {
+        Ip2GeoCache cache = new Ip2GeoCache(1);
+        String datasource = "datasource";
+        Map<String, Object> response1 = new HashMap<>();
+        Map<String, Object> response2 = new HashMap<>();
+        assertNotSame(response1, response2);
```

Comment:
why do we need to assert here?

Reply:
Just to make it looks apparent that `assertSame` won't treat two maps as same even if they have same content.